### ISSUE TITLE
Retrograde memory impairment

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6771]
**Preferred term label:**

**Synonyms**
Retrograde amnesia

**Definition (free text, please give PubMed ID)**
Inability to retrieve information from the long-term memory that was acquired before the onset of amnesia.
PMID:24041667 

**Parent term (use hpo.jax.org/app)**
Long-term memory impairment

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
